allensdk/core/brain_observatory_cache.py

```diff
@@ -551,6 +553,32 @@ def get_ophys_experiment_events(self, ophys_experiment_id, file_name=None):
 
         return np.load(file_name, allow_pickle=False)["ev"]
 
+    def get_eye_tracking(self, ophys_experiment_id) -> np.ndarray:
+        """
+        Download the eye tracking data for an ophys_experiment if it hasn't
+        already been downloaded and return it.
+
+        Parameters
+        ----------
+        ophys_experiment_id
```

Comment:
I assume you confirmed that the ids on the files are experiment_ids and not session_ids as we previously thought?

Reply:
They are experiment ids